sdk/ios/Configsum/Configsum/Configsum/Networking/Logger.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Logger.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-04.
+
+import Foundation
+
+internal struct Logger {
+    internal let environment: Environment
+
+    func log(_ request: URLRequest) {
+        guard let url = request.url else { return }
+        self.log("🌎 URLRequest: \(url)")
```

Comment:
I think the logger should log the not happy scenario as well, I guess the url is composed using the components from the client, if there is something wrong with the URL the client should know, a suggestion would be to use optional chaining with default value 
e.g 
```
let url = requerst.url?.absoluteString ?? "nil" 
self.log("🌎 URLRequest: \(url)")
```